blacklisted_websites.txt

```diff
@@ -1071,3 +1071,22 @@ healthflyup\.com
 supplementschoice\.com
 lumagenexuk\.co\.uk
 garciniacambogialean\.com
+ nutritionfit\.org
+oralhealthplus\.com
+drhelpnutrition\.org
+supplementlab\.org
+supplementstest\.org
+megacleanseradvice\.com
+worldmuscleking\.com
+quicksupplementfact\.com
+maleenhancementshop\.info
+gomusclebuilding\.com
+healthyorder\.org
```

Comment:
Pattern-matched I think is a weaker heuristic than outright blacklisted.  The ones which are already blacklisted should be pruned, I agree.  Not sure if I can fix this today.  Thanks for the feedback!